### DISCUSSION TITLE
Should `quitguard` on CLI and in rc file behave the same?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1092] [Q&A]
I am not sure whether the following is intended or not:

`vd --quitguard file.csv` keeps the quitguard for the "top sheet" even when I go into the columes sheet.

So after the key combo

```
$ vd --quitguard file.csv
shift-C
q
q
```

VisiData blocks with a "quit guarded sheet?" confirmation.

However, after

```
$ echo "options.quitguard = True" > ~/.visisdatarc
$ vd file.csv
q
q
```

VisiData closes without confirmation

If that is expected, why is it so?

**Comment by saulpw** [comment]
The syntax in .visidatarc is Python.  Use `options.quitguard=True`.

**Reply by halloleo** [reply]
Saul, that's what I have in my rc file - it was just a typo in the issue which I now have corrected. Sorry for the confusion!

The problem exists with the correct `.visidatarc`. I am using VisiData 2.4 on CentOS.

**Reply by halloleo** [reply]
Now I tried it on another machine with VisiData 2.5 on macOS and there I cannot get "quitguarding" to work at all! Neither via

    vd --quitguard file.csv

nor  via

    options.quitguard = True

in `~/.visisdatarc`.

**Reply by saulpw** [reply]
You may want to read the CHANGELOG as the behavior of the quitguard has changed in the past couple of versions.  In 2.5 (and future) it only applies if the sheet has been modified.

**Reply by halloleo** [reply]
Oh, interesting! Thanks. 

Any way to get the old behaviour (confirmation on 'q' for the top sheet)?

**Reply by saulpw** [reply]
If you set the option on a sheet specifically, it should guard it regardless of whether it's modified.  But in 2.5 it doesn't, which is a bug.  It'll be fixed soon.
